pkg/git/fetch.go

```diff
@@ -16,58 +30,87 @@ type Fetcher struct {
 }
 
 func (f Fetcher) Fetch(dir, gitURL, gitRevision, metadataDir string) error {
-	f.Logger.Printf("Cloning %q @ %q...", gitURL, gitRevision)
+	f.Logger.Printf("Fetching %q @ %q...", gitURL, gitRevision)
```

Comment:
I don't think changing the ux here is necessary.

Reply:
Sounds good, fixed.